tests/test_tools.sh

```diff
@@ -332,5 +337,38 @@ run_command "gt_mpi_gather -l $WORKSPACE/loader.json -j $QUERY_JSON --produce-Br
 create_query_json $REFERENCE_GENOME
 run_command "gt_mpi_gather -l $WORKSPACE/loader.json -j $QUERY_JSON --produce-Broad-GVCF"
 
+# Test consolidate_genomicsdb_array and gt_mpi_gather after consolidation
+create_sample_list t0.vcf.gz
+ARRAY="1\$1\$249250621"
+
+# Workspace and array specified cannot be null
+EMPTY=""
+run_command "consolidate_genomicsdb_array --workspace $EMPTY --array $ARRAY" ERR
+run_command "consolidate_genomicsdb_array -w $WORKSPACE -a $EMPTY" ERR
+
+run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -o" 1 85 24 85 "#40"
+run_command "consolidate_genomicsdb_array --workspace $WORKSPACE --array $ARRAY"
+run_command "gt_mpi_gather -l $WORKSPACE/loader.json -j $QUERY_JSON --print-AC"
+create_sample_list t1.vcf.gz
+run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -a" 2 85 24 85 "#41"
+run_command "consolidate_genomicsdb_array -w $WORKSPACE -a $ARRAY -p"
+run_command "gt_mpi_gather -l $WORKSPACE/loader.json -j $QUERY_JSON --print-AC"
+
+# Test consolidate_genomicsdb_array with buffer-size
+create_sample_list t2.vcf.gz
+run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -a" 3 85 24 85 "#42"
+run_command "consolidate_genomicsdb_array -w $WORKSPACE -a $ARRAY -z 1024 -p"
+run_command "gt_mpi_gather -l $WORKSPACE/loader.json -j $QUERY_JSON --print-AC"
+
+# Test consolidate_genomicsdb_array with batch-size
+create_sample_list t0.vcf.gz
+run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -o" 1 85 24 85 "#43"
+create_sample_list t1.vcf.gz
+run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -a" 2 85 24 85 "#44"
+create_sample_list t2.vcf.gz
+run_command_and_check_results "vcf2genomicsdb_init -w $WORKSPACE -s $SAMPLE_LIST -a" 3 85 24 85 "#45"
+run_command "consolidate_genomicsdb_array -w $WORKSPACE -a $ARRAY -z 1024 -b 2 -p"
+run_command "gt_mpi_gather -l $WORKSPACE/loader.json -j $QUERY_JSON --print-AC"
```

Comment:
I think it may be worth having a couple of tool tests that include `vcf2genomicsdb` and `consolidate_genomicsdb_array` in the same sequence. Maybe even a simple one that checks that `gt_mpi_gather` returns the same stuff before and after consolidate?

Reply:
Actually, `run_command_and_check_results` runs `vcf2genomicsdb` in addition to `vcf2genomicsdb_init` - see [Line 199.](https://github.com/GenomicsDB/GenomicsDB/blob/b72cc9fae099067125e2fa662a16c9bf3492bc83/tests/test_tools.sh#L199)

Reply:
@mlathara, I do run gt_mpi_gather, but you probably want the results checked against what was returned before consolidation. I will put that check in.